cinq_auditor_required_tags/__init__.py

```diff
@@ -326,30 +331,38 @@ def process_actions(self, actions):
         """
         notices = {}
         notification_contacts = {}
-
+        AWS_EC2_INSTANCE = 1
+        AWS_S3_BUCKET = 5
         try:
             for action in actions:
                 resource = action['resource']
-                if resource.resource_type == 'aws_ec2_instance':
-                    metrics = {"instance_type": resource.instance_type, "public_ip": resource.public_ip}
-
-                elif resource.resource_type == 'aws_s3_bucket':
-                    metrics = {"website_enabled": resource.website_enabled}
-
-                else:
-                    metrics = {}
+                metrics = {}
+                self.log.info('Resource is {} {}'.format(resource, dir(resource)))
+                if resource.resource_type_id == AWS_EC2_INSTANCE:
+                    for prop in resource.properties:
+                        if prop.name == "instance_type":
+                            instance_type = prop.value
+                        if prop.name == "public_ip":
+                            public_ip = prop.value
+                    metrics = {"instance_type": instance_type, "public_ip": public_ip}
```

Comment:
`instance_type` and `public_ip` have the possibility to not be set.

I would change 
```
if resource.resource_type_id == AWS_EC2_INSTANCE:
    for prop in resource.properties:
        if prop.name == "instance_type":
            instance_type = prop.value
        if prop.name == "public_ip":
            public_ip = prop.value
            
    metrics = {"instance_type": instance_type, "public_ip": public_ip}
```
to
```
if resource.resource_type_id == AWS_EC2_INSTANCE:
    instance_type = "No instance type found"
    public_ip = "No public ip found"
    for prop in resource.properties:
        if prop.name == "instance_type":
            instance_type = prop.value
        if prop.name == "public_ip":
            public_ip = prop.value
            
    metrics = {"instance_type": instance_type, "public_ip": public_ip}
```

Reply:
will change to a generic "Not Found"